lib/learn_test/strategies/none.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module LearnTest
+  module Strategies
+    class None < LearnTest::Strategy
+      def service_endpoint
+        '/e/flatiron_none'
+      end
+
+      def detect
+        runner.files.include?('.canvas')
+      end
+
+      def run
+        puts "This directory doesn't appear to have any specs but a submission will be made."
```

Comment:
Is this necessary?

Reply:
That's a good question—my thought was that, because `learn test` usually results in test output, it might be confusing for it to go silent for these cases?

Let me check with curriculum + product to get their preference.